lib/correios/cep/address_finder.rb

```diff
@@ -19,6 +22,21 @@ def web_service
       def parser
         @parser ||= Correios::CEP::Parser.new
       end
+
+      private
+
+      def format_response(response)
+        case response
+        when "CEP NAO ENCONTRADO" 
+          raise CEPNotFound.new "" 
+        when "BUSCA DEFINIDA COMO EXATA, 0 CEP DEVE TER 8 DIGITOS" 
+          raise InvalidCEPFormat.new "" 
+        when "CEP NAO INFORMADO"
+          raise CEPNotFound.new "" 
```

Comment:
Required zip code differs of zip code not found, it should not treated in the same way.